doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -112,6 +113,12 @@ OpenStack Neutron supports open APIs and a pluggable backend where different plu
 
 Neutron supports both core plugins that deal with L2 connectivity and IP address management, and service plugins that support services such as L3 routing, Load Balancers, Firewalls, etc.
 
+<a name="3.2.6"></a>
+### 3.2.6. Acceleration
+NFV Acceleration deals with both hardware and software accelerations. Hardware acceleration is the use of specialized hardware to perform some function faster than is possible by executing the same function on a general-purpose CPU or on a traditional networking (or other I/O) device (e.g. NIC, switch, storage controller, etc.). The hardware accelerator covers the options for ASICs, Smart NIC, FPGAs, GPU etc. to offload the main CPU, and to accelerate workload performance. NFVI should manage the accelerators by plugins and provide the acceleration capacibilities to VNFs.
```

Comment:
Can you change NFV Acceleration -> Acceleration since this sentence talks about acceleration in general.